packages/playground/index.html

```diff
@@ -15,11 +15,29 @@
           >Link copied to clipboard!</span
         >
         <span
-          id="share-button"
+        id="share-reset-button"
+        class="share"
+        title="Click to share the current code"
+        >Reset</span
+      > | 
+        <span
+          id="share-current-button"
           class="share"
           title="Click to share the current code"
-          >Share</span
-        >
+          >Share Current</span
+        > | 
+        <span
+        id="share-open-button"
+        class="share"
+        title="Click to share the current code"
+        >Share Open</span
```

Comment:
This doesn't really seem to work. If I open two different files, the whole workspace seems to be shared anyway. Also, deleting a file and then sharing it with this button seems to reinstate the file.